service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -268,6 +268,7 @@ private void createNewMethod(
             0,
             objectMapper.writeValueAsString(inputs),
             objectMapper.writeValueAsString(outputs),
+            null,
```

Comment:
my only question is: is it ok for this value to be null, instead of an empty string?

I assume the answer is "yes" (because I see RecordType - also a string - was also being set to null) so I'm approving. But I'm curious to understand, if you have any comment

Reply:
I think it should be fine for 2 reasons: 1) any database entries prior to this change will have null user IDs associated, so this column should be accepting of null values, and 2) we probably don't want to associate a user id with a template run set anyways. Tbh, I don't think empty string vs null makes a huge difference, but I stuck with null for consistency with RecordType